arlo-client/src/components/CreateAudit.tsx

```diff
@@ -55,6 +97,32 @@ const CreateAudit = ({ history }: RouteComponentProps<ICreateAuditParams>) => {
       >
         Create a New Audit
       </Button>
+      {elections.elections.length > 0 ? (
+        <>
+          <RadioGroup
```

Comment:
Why radio buttons instead of just a series of links?

Reply:
@eventualbuddha That one's up to @mcchilders . This was how she had it in her mockup. :slightly_smiling_face: